utils/spacewalk-common-channels.ini

```diff
@@ -1296,6 +1296,46 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
 
+[opensuse_micro5_4]
+checksum = sha256
+archs    = x86_64, aarch64
+name     = openSUSE Leap Micro 5.4 (%(arch)s)
+gpgkey_url = https://download.opensuse.org/distribution/leap-micro/5.4/product/repo/Leap-Micro-5.4-x86_64-Media1/gpg-pubkey-3dbdc284-53674dd4.asc
+gpgkey_id = 3DBDC284
+gpgkey_fingerprint = 22C0 7BA5 3417 8CD0 2EFE  22AA B88B 2FD4 3DBD C284
+repo_url = https://download.opensuse.org/distribution/leap-micro/5.4/product/repo/Leap-Micro-5.4-%(arch)s-Media1/
+dist_map_release = 15.5
```

Comment:
15.5? I think it should be 15.4. But I wonder if we better not set this here.
nobody really knows what this is doing and is mostly used for RedHat and Fedora.

Reply:
It was set as 15.4 for version 5.3 so I just guessed that for 5.4 it would be 15.5. If it's not needed, we can just remove it.

Reply:
it would be "5.4" not "15.4".
But I would suggest to remove it.

Reply:
From what I see from the source code of spacewalk-common-channels, the `dist_map_release` is needed only when the `--default-channels` option is used. In that case the API [distchannel.setMapForOrg](https://suma-ref43-srv.mgr.prv.suse.net/rhn/apidoc/handlers/DistChannelHandler.jsp#top) is called, and that values is used as `release`. Not idea what it does, but I guess it will fail without a value.

Reply:
From what I could understand, it should be matching the version of the distribution, so I updated the configuration accordingly.